packages/messaging/src/controllers/sw-controller.ts

```diff
@@ -215,7 +215,7 @@ export class SWController extends ControllerInterface {
       return;
     }
 
-    const notificationInformation = Object.assign({}, msgPayload.notification);
+    const notificationInformation = { ...msgPayload.notification };
```

Comment:
This feels less clear that you are creating a clone of the object.

Reply:
I'll have to disagree, using the spread operator feels more natural to me in this case :)

I'd like to keep this anyway, because spread operator allows for better type checking and inference, so it's more than a style choice in this case.

https://palantir.github.io/tslint/rules/prefer-object-spread/